src/httprpc.cpp

```diff
@@ -18,10 +18,23 @@
 #include <stdio.h>
 
 #include <boost/algorithm/string.hpp> // boost::trim
+#include <boost/signals2/signal.hpp>
 
 /** WWW-Authenticate to present with 401 Unauthorized response */
 static const char* WWW_AUTH_HEADER_DATA = "Basic realm=\"jsonrpc\"";
 
+boost::signals2::signal<void (JSONRPCRequest&, const HTTPRequest&)> PrepareJSONRPCRequestCallbacks;
+
+void RegisterJSONRPCRequestPreparer(const JSONRPCRequestPreparer& preparer)
+{
+    PrepareJSONRPCRequestCallbacks.connect(preparer);
+}
+
+void UnregisterJSONRPCRequestPreparer(const JSONRPCRequestPreparer& preparer)
```

Comment:
This gets never called, right?

Reply:
Right, nor does /wallet/ get unregistered... Fixing